### PR TITLE
[MRG + 1] Fix spelling segmentation

### PR DESCRIPTION
**Pull request by marinacelestine**

### CODE REVIEW

File: sammba/segmentation/brain_mask.py [hunk 1]
```diff
@@ -40,6 +40,46 @@ def brain_extraction_report(head_file, brain_volume, write_dir=None,
...
+        Wether or not to use caching.
+
+    terminal_output : one of {'stream', 'allatonce', 'file', 'none'}
+        Control terminal output :
+            'stream' : displays to terminal immediately,
+            'allatonce' : waits till command is finished to display output,
+            'file' : writes output to file
+            'none' : output is ignored
+
+    digits : int
```
Comment (salma1601) [inline sammba/segmentation/brain_mask.py]: you can add *optional* here

**Comment by salma1601** [comment]
LGTM, apart the minor comment

**Comment by marinacelestine** [comment]
fixes #133 , fixes #134

**Comment by salma1601** [comment]
Ok, merging. Thanks !

**Pull request merged**